views/employees.py

```diff
@@ -102,10 +102,14 @@ def employee_uri():
             if empl_id:
                 for x in empl_id:
```

Comment:
The 'DELETE' action for deleting multiple employees needs to be moved to the employees_uri function. I'll fix it in a separate pull request since it isn't related to your pull request.